packages/contracts/tests/js/launchSequenceTest/DeployZEROContractsTest.js

```diff
@@ -25,10 +25,11 @@ contract('Deploying the ZERO contracts: LCF, CI, ZEROStaking, and ZEROToken ', a
     const coreContracts = await deploymentHelper.deployLiquityCore();
     ZEROContracts = await deploymentHelper.deployZEROContracts(multisig);
     await deploymentHelper.connectZEROContracts(ZEROContracts);
-    await deploymentHelper.connectZEROContractsToCore(ZEROContracts, coreContracts, multisig);
+    await deploymentHelper.connectZEROContractsToCore(ZEROContracts, coreContracts);
 
     zeroStaking = ZEROContracts.zeroStaking;
     zeroToken = ZEROContracts.zeroToken;
+    sovToken = ZEROContracts.zeroToken;
```

Comment:
Maybe we can just rename the current zeroToken to sovToken, but it will lead us to the huge changes for almost all of the contract test, which will make harder to review.
So i'd prefer to do it in the separate PR specific to renaming the variable